sycl/include/CL/sycl/detail/spirv.hpp

```diff
@@ -173,31 +174,26 @@ EnableIfNativeBroadcast<T> GroupBroadcast(T x, id<Dimensions> local_id) {
   }
   using IdT = vec<size_t, Dimensions>;
   using OCLT = detail::ConvertToOpenCLType_t<T>;
+  using WidenedT = conditional_t<
+      std::is_same<OCLT, cl_char>() || std::is_same<OCLT, cl_short>(), cl_int,
+      conditional_t<
+          std::is_same<OCLT, cl_uchar>() || std::is_same<OCLT, cl_ushort>(),
+          cl_uint,
+          conditional_t<std::is_same<OCLT, _Float16>::value, cl_float, OCLT>>>;
```

Comment:
This piece of code should probably outlined into a separate helper, because it is already used twice.

Reply:
Refactored into a helper.